docker/cortx-deploy/Dockerfile

```diff
@@ -61,18 +61,18 @@ RUN yum install yum-utils -y \
 COPY ./*.txt ./${CORTX_IMAGE}/*.txt ./compatibility.info ./generate-release-info.sh /opt/seagate/cortx/
 
 RUN yum install --nogpgcheck -y python3-pip \
-    && pip3 install yq \
+    && pip3 install yq==2.14.0 \
     && while read component; do grep "$component" < /opt/seagate/cortx/python_requirements.txt; done </opt/seagate/cortx/cortx-component-rpms.txt | sed 's/#.*//g' > /opt/seagate/cortx/requirements.txt \
     && while read component; do grep "$component" < /opt/seagate/cortx/python_requirements.ext.txt; done </opt/seagate/cortx/cortx-component-rpms.txt | sed 's/#.*//g' > /opt/seagate/cortx/requirements.ext.txt \
     && pip3 install  --no-cache-dir --trusted-host $(echo $BUILD_URL | awk -F '/' '{print $3}') -i $BUILD_URL/python_deps -r /opt/seagate/cortx/requirements.txt -r /opt/seagate/cortx/requirements.ext.txt \
     && yum clean all && rm -rf /var/cache/yum
 
-RUN if [[ "$CORTX_IMAGE" = "cortx-rgw" ]] || [[ "$CORTX_IMAGE" = "cortx-all" ]] || [[ "$CORTX_IMAGE" = "cortx-data" ]]; then yum --nogpgcheck -y --disablerepo="EOS_Rocky_8_OS_x86_64_Rocky_8" install libfabric-1.11.2; fi \
+RUN if [[ "${CORTX_IMAGE}" = "cortx-rgw" ]] || [[ "${CORTX_IMAGE}" = "cortx-all" ]] || [[ "${CORTX_IMAGE}" = "cortx-data" ]]; then yum --nogpgcheck -y --disablerepo="EOS_Rocky_8_OS_x86_64_Rocky_8" install libfabric-1.11.2; fi \
```

Comment:
Please create CORTX images using the custom-ci pipeline with your repo and branch as input - Refer - https://eos-jenkins.colo.seagate.com/job/GitHub-custom-ci-builds/job/generic/job/custom-ci/
Test CORTX deployment with images generated from the above job.

Reply:
1. Created custom images using jenkins job : https://eos-jenkins.colo.seagate.com/job/GitHub-custom-ci-builds/job/generic/job/custom-ci/
2. After images were build took the latest image and deployed cortx cluster : https://eos-jenkins.colo.seagate.com/job/Cortx-Automation/job/RGW/job/setup-cortx-rgw-cluster/

The Cortx deployment is failing for ha pods.

Reply:
Thanks. I will validate this part. Please freeze any code changes to this PR now.

Reply:
Sure will do that.

Reply:
Deployment is working. Validated with the below steps - 

- Image Generation - https://eos-jenkins.colo.seagate.com/job/GitHub-custom-ci-builds/job/generic/job/custom-ci/6323/console 

```
00:38:02.571  cortx-docker.colo.seagate.com/seagate/cortx-all:2.0.0-6323-custom-ci
00:38:02.571  cortx-docker.colo.seagate.com/seagate/cortx-rgw:2.0.0-6323-custom-ci
00:38:02.571  cortx-docker.colo.seagate.com/seagate/cortx-data:2.0.0-6323-custom-ci
00:38:02.571  cortx-docker.colo.seagate.com/seagate/cortx-control:2.0.0-6323-custom-ci
```

- Deployment -  https://eos-jenkins.colo.seagate.com/job/Cortx-Automation/job/RGW/job/setup-cortx-rgw-cluster/5422/parameters/

![image](https://user-images.githubusercontent.com/63289460/170657205-6b9dd6dc-64c3-4626-8eed-233fd53dc205.png)